pkg/client/cache/securitycontextconstraints.go

```diff
@@ -0,0 +1,43 @@
+package cache
+
+import (
+	kapi "k8s.io/kubernetes/pkg/api"
+	kerrors "k8s.io/kubernetes/pkg/api/errors"
+	"k8s.io/kubernetes/pkg/client/cache"
+	"k8s.io/kubernetes/pkg/controller/framework"
+)
+
+// IndexerToSecurityContextConstraintsLister gives a store List and Exists methods. The store must contain only SecurityContextConstraints.
+type IndexerToSecurityContextConstraintsLister struct {
+	cache.Indexer
+}
+
+// Exists checks if the given SecurityContextConstraints exists in the store.
+func (s *IndexerToSecurityContextConstraintsLister) Exists(scc *kapi.SecurityContextConstraints) (bool, error) {
```

Comment:
I don't see this being used anywhere. We sort of agreed with @deads2k to keep in these libraries only what we need.